frontend/catalyst/pennylane_extensions.py

```diff
@@ -794,7 +791,8 @@ def jvp(f: DifferentiableLike, params, tangents, *, method=None, h=None, argnum=
 
     This function allows the Jacobian-vector Product of a hybrid quantum-classical function to be
     computed within the compiled program. Outside of a compiled function, this function will simply
-    dispatch to its JAX counterpart ``jax.jvp``.
+    dispatch to its JAX counterpart ``jax.jvp``. The function ``f`` can return any pytree-like
+    shape.
```

Comment:
I think my point was more about checking the type signature and description of arguments and results of the gradient functions. For example looking at this one:
```
    Returns (Tuple[Array]):
        Return values of ``f`` paired with the JVP values.
```
I guess the description of it is already super generic, so it could still be considered "correct" but I think it would be better if we actually described what the output looks like.

The return type (which is actually incorrectly formatted here to be picked up by sphinx) is also outdated now since we are no longer restricted to Arrays. I'm not sure about the best type to use in its place, JAX seems to just use `Tuple[Any]`.

Reply:
I have change the signature to tuple(any) and fixed the format issue in VJP and JVP, let me know if that works on your side.